cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
```

Comment:
It is annoying how Cypress continues to add this file... If we don't accept this file, it may just come back in another PR

Reply:
did not even notice :D